### PR TITLE
Address feedback for the syntax.rst

### PR DESCRIPTION
**Pull request by zbraniecki**
@Pike , @stasm - r?

### CODE REVIEW

File: docs/syntax.rst [hunk 1]
```diff
@@ -2,9 +2,14 @@
 The Guide To The FTL Syntax
 ===========================
 
-FTL is an open source, localization-specific scripting language used to process 
-gender, plurals, conjugations, and most of the other quirky elements of natural 
-language.
+FTL is a localization language used for describing translation resources.
```
Comment (stasm) [inline docs/syntax.rst]: Even simpler:

    FTL is a syntax for describing translations.

File: docs/syntax.rst [hunk 2]
```diff
@@ -2,9 +2,14 @@
...
+
+It assigns messages in a given language to unique identifiers used by
+the software to retrieve the message.
+
+FTL is designed to be simple to read, but allow to represent complex concepts
```
Comment (stasm) [inline docs/syntax.rst]: Maybe:

    but at the same time allows to represent...

File: docs/syntax.rst [hunk 3]
```diff
@@ -2,9 +2,14 @@
...
+from natural languages like gender, plurals, conjugations,
+and most of the other quirky elements.
```
Comment (stasm) [inline docs/syntax.rst]: Drop the last part (`and most..`) and:

    from natural languages like genders, plurals, grammatical cases, conjugations and others.

File: docs/syntax.rst [hunk 4]
```diff
@@ -59,6 +64,29 @@ meaningful, FTL allows for special quote delimited strings as can be seen in
...
+
+::
+
+    brandName = Loki
+    installing = Installing { brandName }.
+
+    menu-save = Save
+    help-menu-save = Click "{ menu-save }" to save the file.
+
+In FTL strings may use special syntax to incorporate small pieces of
```
Comment (stasm) [inline docs/syntax.rst]: Let's use this opportunity to fix the wording here:

    Strings in FTL may use...

File: docs/syntax.rst [hunk 5]
```diff
@@ -59,6 +64,29 @@ meaningful, FTL allows for special quote delimited strings as can be seen in
...
+programmable interface. Those pieces are denoted with curly braces ``{`` and
+``}`` and are called placeables.
+
+One example of a placeable is a reference to another entity.
+
+This generally helps to keep certain translations consistent across the
```
Comment (stasm) [inline docs/syntax.rst]: Replace `this` with `Referencing other entities...`.

File: docs/syntax.rst [hunk 6]
```diff
@@ -74,10 +102,9 @@ Interpolation and External Arguments
         "emailCount": 5
     }
 
-In FTL strings may use special syntax to incorporate small pieces of 
-programmable interface. Those pieces are denoted with curly braces ``{`` and 
-``}`` and are called placeables.  The most common use case for a placeable is 
-to put an external argument, provided by the developer, into the string.
+
+The most common use case for a placeable is to put an external argument,
```
Comment (stasm) [inline docs/syntax.rst]: Since we've just mentioned entity references:

    `Another common use case...`

File: docs/syntax.rst (hunk 1 above)
Comment (zbraniecki) [inline docs/syntax.rst]: I prefer what I wrote over this. I took it from Wikipedia articles for HTML and CSS and I believe telling people that it's a language is more appropriate than that it's a syntax.

Comment (stasm) [inline docs/syntax.rst]: OK, I'm not going to block on this but keep in mind that in our domain the word `language` is ambiguous and there's a good chance people will think about the natural language here.  How about `file format`?

Comment (zbraniecki) [inline docs/syntax.rst]: oh, good call! changing :)

**Comment by Pike** [comment]
The whitespace handling still sounds undefined, which it probably isn't.

I think the paragraph about contexts can go away.

The usage of "assign, retrieve, store" feels pretty heavy. Can we get by without any of them?

**Comment by zbraniecki** [comment]
>  The whitespace handling still sounds undefined, which it probably isn't.

Which paragraph it is?
The only paragraph related to white spaces I see is the `Working with text` and it describes what we do with white spaces. Not sure what's undefined about it.

> I think the paragraph about contexts can go away.

There is one in `Interpolation and External Arguments` and one in `Hello World`.
Which one?


> The usage of "assign, retrieve, store" feels pretty heavy. Can we get by without any of them?

There are:
 * 2 `assign`
 * 2 `retrieve`
 * 8 `store`

I'll fix the stores, are you ok with the volume of assign and retrieve?

**Comment by Pike** [comment]
Whitespace is the last paragraph in https://github.com/zbraniecki/l20n.js/blob/256e3991fcf659a955b262ebb65c1c18707b5f3c/docs/syntax.rst#working-with-text-multiline-quote-delimited-strings.

Context paragraph is the last one in https://github.com/zbraniecki/l20n.js/blob/256e3991fcf659a955b262ebb65c1c18707b5f3c/docs/syntax.rst#hello-world.

Thanks for fixing the stores. I'd also look at the assign/retrieve usages, just because I think they're opportunities to use easier language. For example

> FTL is a localization file format used for describing translation resources.
> It assigns messages in a given language to unique identifiers used by the software to retrieve the message.

vs

> FTL is a localization file format used for describing translation resources.
> Software refers to the messages in a given language through unique identifiers.

... though, thinking about that sentence, we should add something that explains that we're doing mono-lingual files?

**Pull request merged**